randovania/data/json_data/prime1.json

```diff
@@ -788,7 +1433,122 @@
                             "connected_dock_index": 1,
                             "dock_type": 0,
                             "dock_weakness_index": 0,
-                            "connections": {}
+                            "connections": {
+                                "Dock to Subchamber Four": {
+                                    "type": "and",
```

Comment:
This is being a big and: `Power Beam and Ice Beam and Wave Beam and Plasma Beam and Missile ≥ 50 and Flamethrower and Charge Beam and Knowledge (Beginner)`